### ISSUE TITLE
Decreased breath sounds

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10188]
**Preferred term label:**
Decreased breath sounds

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0238844

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Chylothorax, hemothorax, pneumothorax

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as

HP:4000214
DIminished breath sounds
syn: Decreased breath sounds
